src/resources/PowerShellWorkerStrings.resx

```diff
@@ -59,8 +59,8 @@
             : using a System.ComponentModel.TypeConverter
             : and then encoded with base64 encoding.
     -->
-  <xsd:schema id="root" xmlns="" xmlns:xsd="https://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
-    <xsd:import namespace="https://www.w3.org/XML/1998/namespace" />
+  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
```

Comment:
Wondering about the "https" -> "http" change... Was this done by a tool automatically?

Reply:
I see: VS 2019 does that. Ok.

Reply:
> I see: VS 2019 does that. Ok.

Ok, so I will leave it as is. Thanks for the code review.